2016-07-05 08:57:46,658 [THD-01][Thread:10][ERROR] DotNetNuke.Services.Exceptions.Exceptions - System.Net.Mail.SmtpException: Failure sending mail. ---> System.IO.IOException: Unable to read data from the transport connection: net_io_connectionclosed.
   at System.Net.Mail.SmtpReplyReaderFactory.ProcessRead(Byte[] buffer, Int32 offset, Int32 read, Boolean readLine)
   at System.Net.Mail.SmtpReplyReaderFactory.ReadLines(SmtpReplyReader caller, Boolean oneLine)
   at System.Net.Mail.SmtpReplyReaderFactory.ReadLine(SmtpReplyReader caller)
   at System.Net.Mail.CheckCommand.Send(SmtpConnection conn, String& response)
   at System.Net.Mail.SmtpTransport.SendMail(MailAddress sender, MailAddressCollection recipients, String deliveryNotify, Boolean allowUnicode, SmtpFailedRecipientException& exception)
   at System.Net.Mail.SmtpClient.Send(MailMessage message)
   --- End of inner exception stack trace ---
   at System.Net.Mail.SmtpClient.Send(MailMessage message)
   at DotNetNuke.Services.Mail.Mail.SendMailInternal(MailMessage mailMessage, String subject, String body, MailPriority priority, MailFormat bodyFormat, Encoding bodyEncoding, IEnumerable`1 attachments, String smtpServer, String smtpAuthentication, String smtpUsername, String smtpPassword, Boolean smtpEnableSSL)
2016-07-05 08:57:47,095 [THD-01][Thread:10][ERROR] DotNetNuke.Services.Exceptions.Exceptions - System.Net.Mail.SmtpException: Failure sending mail. ---> System.IO.IOException: Unable to read data from the transport connection: net_io_connectionclosed.
   at System.Net.Mail.SmtpReplyReaderFactory.ProcessRead(Byte[] buffer, Int32 offset, Int32 read, Boolean readLine)
   at System.Net.Mail.SmtpReplyReaderFactory.ReadLines(SmtpReplyReader caller, Boolean oneLine)
   at System.Net.Mail.SmtpReplyReaderFactory.ReadLine(SmtpReplyReader caller)
   at System.Net.Mail.CheckCommand.Send(SmtpConnection conn, String& response)
   at System.Net.Mail.SmtpTransport.SendMail(MailAddress sender, MailAddressCollection recipients, String deliveryNotify, Boolean allowUnicode, SmtpFailedRecipientException& exception)
   at System.Net.Mail.SmtpClient.Send(MailMessage message)
   --- End of inner exception stack trace ---
   at System.Net.Mail.SmtpClient.Send(MailMessage message)
   at DotNetNuke.Services.Mail.Mail.SendMailInternal(MailMessage mailMessage, String subject, String body, MailPriority priority, MailFormat bodyFormat, Encoding bodyEncoding, IEnumerable`1 attachments, String smtpServer, String smtpAuthentication, String smtpUsername, String smtpPassword, Boolean smtpEnableSSL)
2016-07-05 09:55:21,064 [THD-01][Thread:44][ERROR] DotNetNuke.Services.Exceptions.Exceptions - System.Net.Mail.SmtpException: Failure sending mail. ---> System.IO.IOException: Unable to read data from the transport connection: net_io_connectionclosed.
   at System.Net.Mail.SmtpReplyReaderFactory.ProcessRead(Byte[] buffer, Int32 offset, Int32 read, Boolean readLine)
   at System.Net.Mail.SmtpReplyReaderFactory.ReadLines(SmtpReplyReader caller, Boolean oneLine)
   at System.Net.Mail.SmtpReplyReaderFactory.ReadLine(SmtpReplyReader caller)
   at System.Net.Mail.CheckCommand.Send(SmtpConnection conn, String& response)
   at System.Net.Mail.SmtpTransport.SendMail(MailAddress sender, MailAddressCollection recipients, String deliveryNotify, Boolean allowUnicode, SmtpFailedRecipientException& exception)
   at System.Net.Mail.SmtpClient.Send(MailMessage message)
   --- End of inner exception stack trace ---
   at System.Net.Mail.SmtpClient.Send(MailMessage message)
   at DotNetNuke.Services.Mail.Mail.SendMailInternal(MailMessage mailMessage, String subject, String body, MailPriority priority, MailFormat bodyFormat, Encoding bodyEncoding, IEnumerable`1 attachments, String smtpServer, String smtpAuthentication, String smtpUsername, String smtpPassword, Boolean smtpEnableSSL)
2016-07-05 12:28:59,095 [THD-01][Thread:6][ERROR] DotNetNuke.Services.Exceptions.Exceptions - System.Threading.ThreadAbortException: Thread was being aborted.
   at System.Threading.Thread.AbortInternal()
   at System.Threading.Thread.Abort(Object stateInfo)
   at System.Web.HttpResponse.AbortCurrentThread()
   at DotNetNuke.Modules.Admin.Users.ViewProfile.ProcessQuerystring()
   at DotNetNuke.Modules.Admin.Users.ViewProfile.OnInit(EventArgs e)
   at System.Web.UI.Control.InitRecursive(Control namingContainer)
   at System.Web.UI.Control.AddedControl(Control control, Int32 index)
   at DotNetNuke.UI.Modules.ModuleHost.InjectModuleContent(Control content)
   at System.Web.UI.Control.EnsureChildControls()
   at DotNetNuke.UI.Containers.Container.get_ModuleControl()
   at DotNetNuke.UI.Containers.Container.ProcessModule()
   at DotNetNuke.UI.Skins.Pane.InjectModule(ModuleInfo module)
2016-07-05 12:28:59,330 [THD-01][Thread:6][ERROR] DotNetNuke.Services.Exceptions.Exceptions - DotNetNuke.Services.Exceptions.ModuleLoadException: Unhandled Error Adding Module to contentPane ---> System.Threading.ThreadAbortException: Thread was being aborted.
   at System.Threading.Thread.AbortInternal()
   at System.Threading.Thread.Abort(Object stateInfo)
   at System.Web.HttpResponse.AbortCurrentThread()
   at DotNetNuke.Modules.Admin.Users.ViewProfile.ProcessQuerystring()
   at DotNetNuke.Modules.Admin.Users.ViewProfile.OnInit(EventArgs e)
   at System.Web.UI.Control.InitRecursive(Control namingContainer)
   at System.Web.UI.Control.AddedControl(Control control, Int32 index)
   at DotNetNuke.UI.Modules.ModuleHost.InjectModuleContent(Control content)
   at System.Web.UI.Control.EnsureChildControls()
   at DotNetNuke.UI.Containers.Container.get_ModuleControl()
   at DotNetNuke.UI.Containers.Container.ProcessModule()
   at DotNetNuke.UI.Skins.Pane.InjectModule(ModuleInfo module)
   --- End of inner exception stack trace ---
   at DotNetNuke.UI.Skins.Pane.InjectModule(ModuleInfo module)
   at DotNetNuke.UI.Skins.Skin.InjectModule(Pane pane, ModuleInfo module)
2016-07-05 19:34:44,533 [THD-01][Thread:6][ERROR] DotNetNuke.Services.Exceptions.Exceptions - System.Threading.ThreadAbortException: Thread was being aborted.
   at System.Threading.Thread.AbortInternal()
   at System.Threading.Thread.Abort(Object stateInfo)
   at System.Web.HttpResponse.AbortCurrentThread()
   at DotNetNuke.Modules.Admin.Users.ViewProfile.ProcessQuerystring()
   at DotNetNuke.Modules.Admin.Users.ViewProfile.OnInit(EventArgs e)
   at System.Web.UI.Control.InitRecursive(Control namingContainer)
   at System.Web.UI.Control.AddedControl(Control control, Int32 index)
   at DotNetNuke.UI.Modules.ModuleHost.InjectModuleContent(Control content)
   at System.Web.UI.Control.EnsureChildControls()
   at DotNetNuke.UI.Containers.Container.get_ModuleControl()
   at DotNetNuke.UI.Containers.Container.ProcessModule()
   at DotNetNuke.UI.Skins.Pane.InjectModule(ModuleInfo module)
2016-07-05 19:34:44,611 [THD-01][Thread:6][ERROR] DotNetNuke.Services.Exceptions.Exceptions - DotNetNuke.Services.Exceptions.ModuleLoadException: Unhandled Error Adding Module to contentPane ---> System.Threading.ThreadAbortException: Thread was being aborted.
   at System.Threading.Thread.AbortInternal()
   at System.Threading.Thread.Abort(Object stateInfo)
   at System.Web.HttpResponse.AbortCurrentThread()
   at DotNetNuke.Modules.Admin.Users.ViewProfile.ProcessQuerystring()
   at DotNetNuke.Modules.Admin.Users.ViewProfile.OnInit(EventArgs e)
   at System.Web.UI.Control.InitRecursive(Control namingContainer)
   at System.Web.UI.Control.AddedControl(Control control, Int32 index)
   at DotNetNuke.UI.Modules.ModuleHost.InjectModuleContent(Control content)
   at System.Web.UI.Control.EnsureChildControls()
   at DotNetNuke.UI.Containers.Container.get_ModuleControl()
   at DotNetNuke.UI.Containers.Container.ProcessModule()
   at DotNetNuke.UI.Skins.Pane.InjectModule(ModuleInfo module)
   --- End of inner exception stack trace ---
   at DotNetNuke.UI.Skins.Pane.InjectModule(ModuleInfo module)
   at DotNetNuke.UI.Skins.Skin.InjectModule(Pane pane, ModuleInfo module)
2016-07-05 20:52:04,064 [THD-01][Thread:32][ERROR] DotNetNuke.Services.Exceptions.Exceptions - System.Threading.ThreadAbortException: Thread was being aborted.
   at System.Threading.Thread.AbortInternal()
   at System.Threading.Thread.Abort(Object stateInfo)
   at System.Web.HttpResponse.AbortCurrentThread()
   at DotNetNuke.Modules.Admin.Users.ViewProfile.ProcessQuerystring()
   at DotNetNuke.Modules.Admin.Users.ViewProfile.OnInit(EventArgs e)
   at System.Web.UI.Control.InitRecursive(Control namingContainer)
   at System.Web.UI.Control.AddedControl(Control control, Int32 index)
   at DotNetNuke.UI.Modules.ModuleHost.InjectModuleContent(Control content)
   at System.Web.UI.Control.EnsureChildControls()
   at DotNetNuke.UI.Containers.Container.get_ModuleControl()
   at DotNetNuke.UI.Containers.Container.ProcessModule()
   at DotNetNuke.UI.Skins.Pane.InjectModule(ModuleInfo module)
2016-07-05 20:52:04,173 [THD-01][Thread:32][ERROR] DotNetNuke.Services.Exceptions.Exceptions - DotNetNuke.Services.Exceptions.ModuleLoadException: Unhandled Error Adding Module to contentPane ---> System.Threading.ThreadAbortException: Thread was being aborted.
   at System.Threading.Thread.AbortInternal()
   at System.Threading.Thread.Abort(Object stateInfo)
   at System.Web.HttpResponse.AbortCurrentThread()
   at DotNetNuke.Modules.Admin.Users.ViewProfile.ProcessQuerystring()
   at DotNetNuke.Modules.Admin.Users.ViewProfile.OnInit(EventArgs e)
   at System.Web.UI.Control.InitRecursive(Control namingContainer)
   at System.Web.UI.Control.AddedControl(Control control, Int32 index)
   at DotNetNuke.UI.Modules.ModuleHost.InjectModuleContent(Control content)
   at System.Web.UI.Control.EnsureChildControls()
   at DotNetNuke.UI.Containers.Container.get_ModuleControl()
   at DotNetNuke.UI.Containers.Container.ProcessModule()
   at DotNetNuke.UI.Skins.Pane.InjectModule(ModuleInfo module)
   --- End of inner exception stack trace ---
   at DotNetNuke.UI.Skins.Pane.InjectModule(ModuleInfo module)
   at DotNetNuke.UI.Skins.Skin.InjectModule(Pane pane, ModuleInfo module)
